packages/orchestration/src/service.js

```diff
@@ -252,9 +252,9 @@ const prepareOrchestration = (zone, createChainAccount) =>
          *   the counterparty connection_id
          * @param {IBCConnectionID} controllerConnectionId
          *   self connection_id
-         * @returns {Promise<ChainAccount>}
+         * @returns {Promise<ChainAccountKit['account']>}
```

Comment:
This kind of type seems like a source of confusion rather than help. They well defined type that it should return is a ChainAccount?

Reply:
Added [440fc82](https://github.com/Agoric/agoric-sdk/pull/9312/commits/440fc82285db7a5580f4261c7155863b140393ec) to remove the confusion